applications/external/picopass/picopass_i.h

```diff
@@ -22,13 +22,18 @@
 
 #include "scenes/picopass_scene.h"
 #include "views/dict_attack.h"
+#include "views/loclass.h"
 
 #include <storage/storage.h>
 #include <lib/toolbox/path.h>
 #include <picopass_icons.h>
 
 #define PICOPASS_TEXT_STORE_SIZE 128
 
+#define LOCLASS_NUM_CSNS 9
```

Comment:
I was asking myself if this should be in some sort of "loclass" header, but I realize we use the term "loclass" for both the library as well as the attack, so I think this is a good place for the attack specific `define`s since there aren't many